### PR TITLE
NEU genesis allocation

### PR DESCRIPTION
**Pull request by rhuttman**

### CODE REVIEW

File: node/src/chain_spec.rs [hunk 1]
```diff
@@ -294,3 +294,137 @@ fn testnet_genesis(
...
+	properties.insert("ss58Format".into(), SS_58_FORMAT.into());
+
+	ChainSpec::from_genesis(
+		// Name
+		"Neumann Network",
+		// ID
+		"neumann",
+		ChainType::Live,
+		move || {
+			let allocation_json = &include_bytes!("../../distribution/neumann_alloc.json")[..];
```
Comment (irsal) [inline node/src/chain_spec.rs:312]: does this get prioritized over the 3 big accounts? ie will each of the 3 accounts below (yours, mine and chris') have < 333.33MM NEU?

File: distribution/neumann_alloc.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	[
+		"5CM2JyPHnbs81Cu8GzbraqHiwjeNwX3c9Rr5nXkJfwK9fwrk",
+		3333333333333333333
+	],
+    [
+		"5GcD1vPdWzBd3VPTPgVFWL9K7b27A2tPYcVTJoGwKcLjdG5w",
+        3333333333333333333
+	],
+    [
+        "5DhqAmCUoY7FiMCY7YpTyGT4fFpdXYmYjzzE9fHJwLDyBafZ",
```
Comment (justinzhou93) [inline distribution/neumann_alloc.json:15]: how did you split this into 4 accounts with these splits? i can't see how you're inserting the ratios in the chainspec.rs file below

File: node/src/chain_spec.rs (hunk 1 above)
Comment (rhuttman) [reply node/src/chain_spec.rs:312]: I am going to take the output from subquery and transform it into the allocation json. Our accounts will be in there with whatever amount we have now. We only use this json for endowed accounts

File: distribution/neumann_alloc.json (hunk 2 above)
Comment (rhuttman) [reply distribution/neumann_alloc.json:15]: I made them up as tests. The real one will come later

Comment (justinzhou93) [reply distribution/neumann_alloc.json:15]: ah yah totally, i just meant i can't seem to find where you defined the ratios in the code

**Review by justinzhou93 — APPROVED** [review]
code/results lgtm, but i'll def defer to irsal on whether this works, since she's doing some testing

**Review by irsal — APPROVED** [review]
💪

**Pull request merged**